src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,34 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
+description: Setting up the robot for robocup
+slug: /guides/main/robocup-setup
+authors:
+    - Cameron Murtagh (@CMurtagh-LGTM)
+---
+
+## Networking
+
+1. On the robot with a monitor and keyboard, not via `ssh`.
+2. Rename `system/nugusX/etc/systemd/network/40-wlp58s0-robocup.network`
+   to `system/nugusX/etc/systemd/network/30-wlp58s0.network`
+3. Rename `system/defualt/etc/systemd/30-wlp58s0.network` to `system/default/etc/systemd/40-wlp58s0-robocup.network`
```

Comment:
```suggestion
3. Rename `system/default/etc/systemd/30-wlp58s0.network` to `system/default/etc/systemd/40-wlp58s0-robocup.network`
```
You don't actually have to do this step though. You can just rename the robocup one to the 30 one

Also should really do this in the codebase on a RoboCup branch so it persists and gets set every time.
I know you mention this later but I think that should be the normal way of dealing with robocup.
Have the config setup on a RoboCup branch for each robot and just run from there

Reply:
I think it is clearer if we do *something* with the default one.